app/domain/authentication/authn_k8s/errors.rb

```diff
@@ -0,0 +1,53 @@
+module Authentication
+  module AuthnK8s
+
+    AuthenticatorNotFound = ::Util::ErrorClass.new(
+      "'{0}' wasn't in the available authenticators"
```

Comment:
Were you able to fix these template replacements not working?

Reply:
Yes remember the fix was `ErrorWhatever.new(arg1, arg2)`

Reply:
I've since it fail more recently than that fix though, I think...I'll double check.